dot/peerset/peerstate.go

```diff
@@ -57,8 +57,8 @@ type Info struct {
 
 // node represents state of a single node that we know about
 type node struct {
-	// list of Set the node belongs to.
-	// always has a fixed size equal to the one of PeersState Set. The various possible Set
+	// list of Set, the node belongs to.
+	// always has a fixed size, equal to the one of PeersState Set. The various possible Set
```

Comment:
A bit out of scope, but this is confusing too

Reply:
i think, it says the list is of fix size. we have four types of membership state, so there would be four sets.

Reply:
Maybe rephrase to

```suggestion
	// It always has the fixed size of the PeersState set. The various possible sets
```

Reply:
it would rather keep this as it is. `Set` has a special meaning here.